hack/publish.sh

```diff
@@ -57,27 +46,6 @@ fi
 # overwrite the tag name into values yaml
 sed -i "s/latest/${VOLCANO_IMAGE_TAG}/g" ${RELEASE_FOLDER}/helm/chart/volcano/values.yaml
 
-if [[ "${DOCKER_USERNAME}xxx" == "xxx" ]];then
-  if [[ "${DOCKER_PASSWORD}xxx" == "xxx" ]];then
-    echo "docker username or password not found, quit uploading images"
-    exit 0
-  fi
-fi
-
-echo "${DOCKER_PASSWORD}" | docker login -u "${DOCKER_USERNAME}" --password-stdin
-echo "pushing ${IMAGE_PREFIX}/vc-controller-manager:${VOLCANO_IMAGE_TAG}"
-docker tag ${IMAGE_PREFIX}/vc-controller-manager:${VOLCANO_IMAGE_TAG} ${IMAGE_PREFIX}/vc-controllers:${VOLCANO_IMAGE_TAG}
-docker push ${IMAGE_PREFIX}/vc-controllers:${VOLCANO_IMAGE_TAG}
-docker push ${IMAGE_PREFIX}/vc-controller-manager:${VOLCANO_IMAGE_TAG}
-
-echo "pushing ${IMAGE_PREFIX}/vc-scheduler:${VOLCANO_IMAGE_TAG}"
-docker push ${IMAGE_PREFIX}/vc-scheduler:${VOLCANO_IMAGE_TAG}
-
-echo "pushing ${IMAGE_PREFIX}/vc-webhook-manager:${VOLCANO_IMAGE_TAG}"
-docker tag ${IMAGE_PREFIX}/vc-webhook-manager:${VOLCANO_IMAGE_TAG} ${IMAGE_PREFIX}/vc-admission:${VOLCANO_IMAGE_TAG}
-docker push ${IMAGE_PREFIX}/vc-admission:${VOLCANO_IMAGE_TAG}
-docker push ${IMAGE_PREFIX}/vc-webhook-manager:${VOLCANO_IMAGE_TAG}
-
 echo "Generate release tar files"
 cd ${RELEASE_FOLDER}/
 tar -zcvf volcano-${VOLCANO_IMAGE_TAG}-${OSTYPE}.tar.gz *
```

Comment:
do you know why we need that?

Reply:
I am not sure. I guess for [this](https://github.com/volcano-sh/volcano/archive/refs/tags/v1.6.0.tar.gz)